compiler/src/latex_printer.ml

```diff
@@ -447,5 +447,14 @@ let rec pp_pitem fmt pi =
      closebrace fmt ()
   | PTypeAlias (id,ty) -> pp_typealias fmt id ty (**)
 
+let pp_info fmt =
+  F.fprintf fmt "@[<v>@[The produced LATEX snippet is meant to be included in a@] @ ";
+  F.fprintf fmt "@[jasmincode environment provided by the jasmin package@] @ ";
+  F.fprintf fmt "@[defined in file: @] @ ";
+  F.fprintf fmt "@[https://github.com/jasmin-lang/jasmin/wiki/resources/jasmin.sty@]@]";
+  F.pp_print_newline fmt ();
+  F.pp_print_newline fmt ()
```

Comment:
Are these TEX comment?

Reply:
No. Since the output is not valid Tex (initially), I thought this is not required.

Reply:
I add TeX comments.